go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/cf-lookup-route
```

Comment:
The module should be:

```suggestion
module github.com/cloudfoundry/cf-lookup-route
```

Reply:
Sorry, have fixed that in the other branch already